gg/db.py

```diff
@@ -0,0 +1,30 @@
+import pymongo
+
+
+def db_get_collection():
```

Comment:
can we get it to read the uri from an environment file instead?

Reply:
Yes, we now use python-dotenv to read the URI from an environment file.